ui/src/components/Section/section.scss

```diff
@@ -53,6 +56,9 @@
 
   &.section--theme__funky {
     --sectionBgColor: var(--funkyGradient);
+    .card {
```

Comment:
there might be times when we wouldnt want the card to have that elevation with a funky bg, instead lets just put the card--elevation-3 class on the card we are using